src/Core/Vipr.Core/CodeModel/Vocabularies/Capabilities/OdcmInsertCapability.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Vipr.Core.CodeModel.Vocabularies.Capabilities
+{
+    public class OdcmInsertCapability : OdcmBooleanCapability
+    {
+        public override string TermName
+        {
+            get { return "Org.OData.Capabilities.V1.InsertRestrictions"; }
+        }
+
+        /// <summary>
+        /// Entities can be inserted
+        /// </summary>
+        public bool Insertable
+        {
+            get { return BooleanValue; }
+        }
+
+        public OdcmInsertCapability()
+        {
+            BooleanValue = true;
```

Comment:
Is the value always initialized to true for BooleanCapabilities? Should we set the value in the base class, then?